easy-ui-react/src/Button/Button.stories.mdx

```diff
@@ -0,0 +1,367 @@
+import { Canvas, Meta, Story, ArgsTable } from "@storybook/addon-docs";
+import { createLabelledOptionsControl } from "../utilities/storybook";
+import { Button } from "./Button";
+import { Icon } from "../Icon";
+import AddIcon from "@easypost/easy-ui-icons/Add";
+import CheckCircleIcon from "@easypost/easy-ui-icons/CheckCircle";
+import InfoIcon from "@easypost/easy-ui-icons/Info";
+import ArrowBackIcon from "@easypost/easy-ui-icons/ArrowBack";
+
+<Meta
+  title="Button"
+  argTypes={{
+    children: {
+      control: "text",
+    },
+  }}
+  component={Button}
+/>
+
+export const Template = (args) => <Button {...args} />;
+
+# Button
+
+<Canvas>
+  <Story name="Default" args={{}}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+Filled buttons
+
+<Canvas>
+  <Story
+    name="Secondary"
```

Comment:
is it possible to organize these stories under certain groupings to clean up the sidebar?

`name="Filled/Secondary"`

can't remember if it supports nesting at this level